### PR TITLE
Add lldb engine to attach on SSH

### PR DESCRIPTION
**Pull request by paulmaybee**

### CODE REVIEW

File: src/MIDebugEngine/Microsoft.MIDebugEngine.pkgdef [hunk 1]
```diff
@@ -55,6 +55,37 @@
...
 
+; Registration to use lldb with the port suppliers
+[$RootKey$\AD7Metrics\Engine\{5D630903-189D-4837-9785-699B05BEC2A9}]
+@="MI Debug Engine - lldb"
+"Attach"=dword:00000001
+; "AddressBP"=dword:00000001
+"AlwaysLoadLocal"=dword:00000001
+; "CallStackBP"=dword:00000001
+; "ConditionalBP"=dword:00000001
+"DataBP"=dword:00000001
```
Comment (WardenGnaw) [inline src/MIDebugEngine/Microsoft.MIDebugEngine.pkgdef:66]:
```suggestion
"DataBP"=dword:00000000
```
`BreakWatch` is not implemented for `lldb`.

https://github.com/microsoft/MIEngine/blob/69512399f0278d37d96a2b91d4818cf1c2ecd549/src/MICore/CommandFactories/MICommandFactory.cs#L501

File: src/MIDebugEngine/Microsoft.MIDebugEngine.pkgdef [hunk 2]
```diff
@@ -55,6 +55,37 @@
...
+"Exceptions"=dword:00000001
+; "FunctionBP"=dword:00000001
+; "HitCountBP"=dword:00000000
+; "JustMyCodeStepping"=dword:00000001
+"Name"="Native (LLDB)"
+"SetNextStatement"=dword:00000001
+; "SuspendThread"=dword:00000001
+"CLSID"="{0fc2f352-2fc1-4f80-8736-51cd1ab28f16}"
+"GlobalVisualizersDirectory"="$PackageFolder$"
+"BreakWhenValueChanges"=dword:00000001
```
Comment (WardenGnaw) [inline src/MIDebugEngine/Microsoft.MIDebugEngine.pkgdef:78]:
```suggestion
"BreakWhenValueChanges"=dword:00000000
```

Part of Data Breakpoints

File: src/MIDebugEngine/Microsoft.MIDebugEngine.pkgdef [hunk 3]
```diff
@@ -55,6 +55,37 @@
...
+"DataBPAsString"=dword:00000001
```
Comment (WardenGnaw) [inline src/MIDebugEngine/Microsoft.MIDebugEngine.pkgdef:79]:
```suggestion
"DataBPAsString"=dword:00000000
```

Part of Data Breakpoints

**Review by WardenGnaw — CHANGES_REQUESTED** [review]

File: src/SSHDebugPS/AD7/AD7UnixAsyncShellCommand.cs [hunk 4]
```diff
@@ -62,7 +62,7 @@ protected override void OnOutputReceived(object sender, string e)
 
                 if (_startCommand != null)
                 {
-                    if (line.EndsWith(_startCommand, StringComparison.Ordinal))
+                    if (line.Contains(_startCommand))
```
Comment (gregg-miskelly) [inline src/SSHDebugPS/AD7/AD7UnixAsyncShellCommand.cs:65]: What does this wind up coming back as? Might be worth a comment.

**Review by gregg-miskelly — APPROVED** [review]
Otherwise LGTM

**Review by WardenGnaw — APPROVED** [review]

**Pull request merged**